graphql/resolvers.go

```diff
@@ -870,6 +870,17 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) HostEvents(ctx context.Context, limit *int, page *int, hostID string, hostTag string) (*HostEvents, error) {
+	events, err := event.FindPaginated(event.AllLogCollection, event.MostRecentHostEvents(hostID, hostTag, *limit), *page, *limit)
```

Comment:
It'd be silly, but since the client can pass null for limit and page can you null check them before dereferencing? Or a better approach might be to make them non-nullable